app/mailers/application_mailer.rb

```diff
@@ -10,12 +10,16 @@ class ApplicationMailer < ActionMailer::Base
   before_action :use_custom_host
 
   default(
-    from: -> { email_from("Community") },
+    from: -> { email_from },
     template_path: ->(mailer) { "mailers/#{mailer.class.name.underscore}" },
   )
 
-  def email_from(topic)
-    "#{SiteConfig.community_name} #{topic} <#{SiteConfig.email_addresses[:default]}>"
+  def email_from(topic = "")
+    if topic.present?
+      "#{SiteConfig.community_name} #{topic} <#{SiteConfig.email_addresses[:default]}>"
+    else
+      "#{SiteConfig.community_name} <#{SiteConfig.email_addresses[:default]}>"
+    end
```

Comment:
```suggestion
  def email_from(topic = "")
    community_name = topic.present? ? "#{SiteConfig.community_name} #{topic}" : SiteConfig.community_name
    
    "#{community_name} <#{SiteConfig.email_addresses[:default]}>"
    end
```
A slightly cleaner way to write this -- since you know that you are always going to have `SiteConfig.community_name`, you could conditionally tack on the `topic` if it is present 😃

Reply:
I **la, la, love** this! Thank you for the suggestion--this is great! ✨